src/backend/web/routes/feeds.js

```diff
@@ -51,4 +52,26 @@ feeds.get('/:id', async (req, res) => {
   }
 });
 
+feeds.post('/', protect, async (req, res) => {
+  const feedData = req.body;
+  if (!feedData.url && !feedData.author) {
```

Comment:
This won't cover every case (i.e., this will only get triggered if both are missing, but not one or the other).  You probably want to say, `if(!(feedData.url && feedData.author)) {`

Reply:
Updated.